src/More/ReportIssueForm.tsx

```diff
@@ -0,0 +1,255 @@
+import React, { useEffect, useState, FunctionComponent } from "react"
+import { useTranslation } from "react-i18next"
+import {
+  ActivityIndicator,
+  Alert,
+  KeyboardAvoidingView,
+  StyleSheet,
+  Platform,
+  TextInput,
+  View,
+  Keyboard,
+  SafeAreaView,
+} from "react-native"
+import { useNavigation } from "@react-navigation/native"
+
+import { GlobalText } from "../components/GlobalText"
+import { Button } from "../components/Button"
+import { reportAnIssue, ReportIssueError } from "./zendeskAPI"
+
+import { Spacing, Layout, Forms, Colors, Outlines, Typography } from "../styles"
+
+const defaultErrorMessage = " "
+const EMAIL_REGEX = RegExp(/\S+@\S+\.\S+/)
```

Comment:
should we be rolling our own email validation? is this something the zendesk api could handle?

Reply:
It does, actually, we could potentially gather the feedback from the errors and then display it back, not sure about the localization aspect of it. Maybe is worth exploring that.

Reply:
@johnschoeman updated that logic